pydax/_dataset.py

```diff
@@ -109,27 +113,43 @@ def _pydax_dir_(self) -> pathlib.Path:
 
     @property
     def _pydax_dir(self) -> pathlib.Path:
-        "Same as :attr:`_pydax_dir`, but create it if it does not exist."
+        "Same as :attr:`_pydax_dir_`, but create it if it does not exist."
         if not self._pydax_dir_.exists():
             self._pydax_dir_.mkdir(parents=True)
         elif not self._pydax_dir_.is_dir():  # pydax_dir exists and is not a directory
             raise NotADirectoryError(f'"{self._pydax_dir_}" exists and is not a directory.')
         return self._pydax_dir_
 
     @property
-    def _file_list_file(self) -> pathlib.Path:
+    def _file_list_file_(self) -> pathlib.Path:
         "Path to the file that stores the list of files in the downloaded dataset."
+        return self._pydax_dir_ / 'files.list'
+
+    @property
+    def _file_list_file(self) -> pathlib.Path:
+        "Same as :attr:`_file_list_file_`, but create the parent directory if it does not exist."
         return self._pydax_dir / 'files.list'
 
-    def download(self) -> None:
+    def download(self,
+                 check: bool = True) -> None:
         """Downloads, extracts, and removes dataset archive. It adds a directory write lock during execution.
 
-        :raises NotADirectoryError: :attr:`Dataset._data_dir` (passed in via ``data_dir`` in the constructor
-                                    :class:`Dataset`) points to an existing file that is not a directory.
+        :param check: Check that the data files were not previously downloaded to :attr:`Dataset._data_dir` (passed in
+            via ``data_dir`` in the constructor :class:`Dataset`) by running :meth:`.is_downloaded`.
```

Comment:
There's a slight ambiguity here. What's the effect of the check being positive/negative?